content/upload/upload_1.md

```diff
@@ -0,0 +1,413 @@
+
+SELECT
+	APP_ID,
+	PLATFORM,
+	ETL_TSTAMP,
+	COLLECTOR_TSTAMP,
+	DVCE_CREATED_TSTAMP,
+	EVENT,
+	EVENT_ID,
+	TXN_ID,
+	NAME_TRACKER,
+	V_TRACKER,
+	V_COLLECTOR,
+	V_ETL,
+	USER_ID,
+	USER_IPADDRESS,
+	USER_FINGERPRINT,
+	DOMAIN_USERID,
+	DOMAIN_SESSIONIDX,
+	NETWORK_USERID,
+	GEO_COUNTRY,
+	GEO_REGION,
+	GEO_CITY,
+	GEO_ZIPCODE,
+	GEO_LATITUDE,
+	GEO_LONGITUDE,
+	GEO_REGION_NAME,
+	IP_ISP,
+	IP_ORGANIZATION,
+	IP_DOMAIN,
+	IP_NETSPEED,
+	PAGE_URL,
+	PAGE_TITLE,
+	PAGE_REFERRER,
+	PAGE_URLSCHEME,
+	PAGE_URLHOST,
+	PAGE_URLPORT,
+	PAGE_URLPATH,
+	PAGE_URLQUERY,
+	PAGE_URLFRAGMENT,
+	REFR_URLSCHEME,
+	REFR_URLHOST,
+	REFR_URLPORT,
+	REFR_URLPATH,
+	REFR_URLQUERY,
+	REFR_URLFRAGMENT,
+	REFR_MEDIUM,
+	REFR_SOURCE,
+	REFR_TERM,
+	MKT_MEDIUM,
+	MKT_SOURCE,
+	MKT_TERM,
+	MKT_CONTENT,
+	MKT_CAMPAIGN,
+	SE_CATEGORY,
+	SE_ACTION,
+	SE_LABEL,
+	SE_PROPERTY,
+	SE_VALUE,
+	TR_ORDERID,
+	TR_AFFILIATION,
+	TR_TOTAL,
+	TR_TAX,
+	TR_SHIPPING,
+	TR_CITY,
+	TR_STATE,
+	TR_COUNTRY,
+	TI_ORDERID,
+	TI_SKU,
+	TI_NAME,
+	TI_CATEGORY,
+	TI_PRICE,
+	TI_QUANTITY,
+	PP_XOFFSET_MIN,
+	PP_XOFFSET_MAX,
+	PP_YOFFSET_MIN,
+	PP_YOFFSET_MAX,
+	REPLACE(USERAGENT, '\"', '') as USERAGENT,
+	BR_NAME,
+	BR_FAMILY,
+	BR_VERSION,
+	BR_TYPE,
+	BR_RENDERENGINE,
+	BR_LANG,
+	BR_FEATURES_PDF,
+	BR_FEATURES_FLASH,
+	BR_FEATURES_JAVA,
+	BR_FEATURES_DIRECTOR,
+	BR_FEATURES_QUICKTIME,
+	BR_FEATURES_REALPLAYER,
+	BR_FEATURES_WINDOWSMEDIA,
+	BR_FEATURES_GEARS,
+	BR_FEATURES_SILVERLIGHT,
+	BR_COOKIES,
+	BR_COLORDEPTH,
+	BR_VIEWWIDTH,
+	BR_VIEWHEIGHT,
+	OS_NAME,
+	OS_FAMILY,
+	OS_MANUFACTURER,
+	OS_TIMEZONE,
+	DVCE_TYPE,
+	DVCE_ISMOBILE,
+	DVCE_SCREENWIDTH,
+	DVCE_SCREENHEIGHT,
+	DOC_CHARSET,
+	DOC_WIDTH,
+	DOC_HEIGHT,
+	TR_CURRENCY,
+	TR_TOTAL_BASE,
+	TR_TAX_BASE,
+	TR_SHIPPING_BASE,
+	TI_CURRENCY,
+	TI_PRICE_BASE,
+	BASE_CURRENCY,
+	GEO_TIMEZONE,
+	MKT_CLICKID,
+	MKT_NETWORK,
+	ETL_TAGS,
+	DVCE_SENT_TSTAMP,
+	REFR_DOMAIN_USERID,
+	REFR_DVCE_TSTAMP,
+	DOMAIN_SESSIONID,
+	DERIVED_TSTAMP,
+	EVENT_VENDOR,
+	EVENT_NAME,
+	EVENT_FORMAT,
+	EVENT_VERSION,
+	EVENT_FINGERPRINT,
+	TRUE_TSTAMP,
+	LOAD_TSTAMP,
+	PARSE_JSON(REPLACE(REPLACE(CONTEXTS_COM_SNOWPLOWANALYTICS_SNOWPLOW_UA_PARSER_CONTEXT_1,'\"', ''),'''','\"')) as CONTEXTS_COM_SNOWPLOWANALYTICS_SNOWPLOW_UA_PARSER_CONTEXT_1,
+	PARSE_JSON(REPLACE(REPLACE(CONTEXTS_COM_SNOWPLOWANALYTICS_SNOWPLOW_WEB_PAGE_1,'\"', ''),'''','\"')) as CONTEXTS_COM_SNOWPLOWANALYTICS_SNOWPLOW_WEB_PAGE_1,
+	PARSE_JSON(REPLACE(REPLACE(CONTEXTS_COM_IAB_SNOWPLOW_SPIDERS_AND_ROBOTS_1,'\"', ''),'''','\"')) as CONTEXTS_COM_IAB_SNOWPLOW_SPIDERS_AND_ROBOTS_1,
+	PARSE_JSON(REPLACE(REPLACE(CONTEXTS_NL_BASJES_YAUAA_CONTEXT_1,'\"', ''),'''','\"')) as CONTEXTS_NL_BASJES_YAUAA_CONTEXT_1
+
+FROM ATOMIC.SAMPLE_EVENTS_BASE )
+
+```
+
+***
+
+#### **Step 5:**  Drop the **SAMPLE_EVENTS_BASE** table
+
+```sql
+DROP TABLE TARGET_DB.ATOMIC.SAMPLE_EVENTS_BASE
+```
+Now you should have the ATOMIC.SAMPLE_EVENTS created and loaded with sample data.
```

Comment:
Change to
"You will now have the ATOMIC.SAMPLE_EVENTS created and loaded with sample data."